app/src/openxr/cpp/OpenXRGestureManager.h

```diff
@@ -45,7 +45,12 @@ XrHandTrackingAimStateFB mFBAimState;
 
 class OpenXRGestureManagerHandJoints : public OpenXRGestureManager {
 public:
-OpenXRGestureManagerHandJoints(HandJointsArray& handJoints);
+typedef struct OneEuroFilterParams {
+float mincutoff;
+float beta;
+float dcutoff;
+} OneEuroFilterParams;
+OpenXRGestureManagerHandJoints(HandJointsArray&, OneEuroFilterParams*);
```

Comment:
Have you considered nullptr as default value for OneEuroFilterParams ? 

Also, it seems we only need the pointer for implementing the nullptr codepath; wouldn't be std::optional more appropriated for that purpose ?

Reply:
The default value is a good suggestion indeed.

WRT using optional for the argument of a function, I think it does not provide anything over a raw pointer. I do see the benefits in a method returning a value but for this case I think the raw pointer is simpler.